sycl/test-e2e/ESIMD/local_accessor_block_load_store.cpp

```diff
@@ -0,0 +1,115 @@
+//==-- local_accessor_block_load_store.cpp  - DPC++ ESIMD on-device test --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// local_accessors are not supported in esimd_emulator yet.
+// UNSUPPORTED: esimd_emulator
+//
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+//
+// TODO: Enable the test when GPU driver is ready/fixed.
```

Comment:
Nit: Can we put the `UNSUPPORTED` and `XFAIL` lines together. Also we might want to consider changing the `XFAIL` to unsupported so CI doesn't break when the GPU driver is updated

Also, the GPU driver was updated yesterday, so maybe it works now?

Reply:
yeah its a hard problem, break CI for everyone once it starts working, or have no way to know it started working, i don't know what the best practice is

Reply:
we had that issue today with thread_id_test that was an example of this and somehow they missed it